CMakeLists.txt

```diff
@@ -1,6 +1,28 @@
-cmake_minimum_required(VERSION 3.8)
+cmake_minimum_required(VERSION 3.11)
 project(ccls LANGUAGES CXX)
 
+### Colored output
+if(NOT WIN32)
+  string(ASCII 27 Esc)
+  set(ColourReset "${Esc}[m")
+  set(ColourBold  "${Esc}[1m")
+  set(Red         "${Esc}[31m")
```

Comment:
This doesn't need to be a full listing. Listing what are used should be good enough

Reply:
removed